msmbuilder/msm/validation/bootstrapmsm.py

```diff
@@ -67,6 +82,11 @@ class BootStrapMarkovStateModel(_MappingTransformMixin):
     of workers making it capable of parallelizing across
     compute nodes via mpi or ipyparallel. This can lead to
     a significant speed up for larger number of samples.
+
+    Example
```

Comment:
I think this is the canonical list of allowed sections

https://github.com/numpy/numpydoc/blob/master/numpydoc/docscrape.py#L105